### ISSUE TITLE
video from sequece of images fails with unknown error

### ISSUE THREAD

**Issue by sanromd** [kkroening/ffmpeg-python#254]
HI @kkroening 

I am creating a set of videos from a high speed FLIR camera. I tried using the code in the examples, but I am getting an unknown error when running the code. I tried running the ffmpeg code directly from the terminal and it works, so I am not sure what I am doing wrong here. Any help will be much appreciated.

Thank you

```python
import ffmpeg, os

process = (
    ffmpeg
    .input('"' + os.path.join(figpath,'*.png')+'"', pattern_type='glob', framerate=f_rate)
    .output(outfile, crf=25, preset='medium', pix_fmt='yuv420p', vcodec='libx264')
)

print(process.get_args())

#['-framerate', '349', '-pattern_type', 'glob', '-i', '"/media/sanromd/data/lab/data/laser_experiments/_unsorted/irdata/laser-experiments/perforation/un_007/test_02/figs/*.png"', '-crf', '25', '-pix_fmt', 'yuv420p', '-preset', 'medium', '-vcodec', 'libx264', '/media/sanromd/data/lab/data/laser_experiments/_unsorted/irdata/laser-experiments/perforation/un_007/test_02/vids/un_007_1.mp4']

process.run(capture_stderr=True, capture_stdout=True)
```

error:

---------------------------------------------------------------------------
Error                                     Traceback (most recent call last)
<ipython-input-66-5c15ec1bac16> in <module>
----> 1 process.run(capture_stderr=True, capture_stdout=True)

/sandbox/opt/share/anaconda/lib/python3.7/site-packages/ffmpeg/_run.py in run(stream_spec, cmd, capture_stdout, capture_stderr, input, quiet, overwrite_output)
    323     retcode = process.poll()
    324     if retcode:
--> 325         raise Error('ffmpeg', out, err)
    326     return out, err
    327 

Error: ffmpeg error (see stderr output for detail)

**Comment by 153957** [comment]
I do not think you need the extra `"` around the input string, just `os.path.join(figpath,'*.png')` should also be fine.
